assembl/views/auth/views.py

```diff
@@ -806,15 +804,17 @@ def request_password_change(request):
         else:
             error = error or localizer.translate(_("This user cannot be found"))
     if error or not user:
-        slug = request.matchdict.get('discussion_slug', None)
-
-        return dict(
-            get_default_context(request),
-            error=error,
-            user_id=user_id,
-            identifier=identifier,
-            slug_prefix="/" + slug if slug else "",
-            title=localizer.translate(_('I forgot my password')))
+        # This is not how to do this!
```

Comment:
OK, that was unhelpful, apologies. We were working closely together and adding notes for future work rather than thinking how it could be received.
What I saw you were trying to do is to check the provenance (backbone/react main view), and change the login view accordingly. I do not believe that auth view technology should be determined by main view technology, but we should have a switch to use one or the other auth view globally, independent of which main view we come from.
We did not implement a switch but changed everything globally, btw. Not the cleanest, but we were working under absurd time constraints. In my view, we did 85% of the work, and many of your valid comments reflect that.